src/Sentry/buildTransitive/Sentry.props

```diff
@@ -32,4 +32,27 @@
               ItemName="FileWrites" />
     </WriteCodeFragment>
   </Target>
+
+  <Target Name="UploadSymbolsToSentry" AfterTargets="AfterBuild" Condition="'$(PkgSentry)' != '' Or '$(SentryCLIDirectory)' != ''">
```

Comment:
On Gradle we only upload on Release builds by default. This makes the dev loop faster and avoid uploading tons of symbols that won't go to production.
We did have an option to opt-in debug build uploads

Reply:
Added `SentryUploadSymbols` MSBuild property.  It's true by default for release builds, and false by default otherwise.  End user can set it in their csproj to override if desired.